build.gradle

```diff
@@ -109,12 +110,48 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allJava
 }
 
+task javadocJar(type: Jar) {
+   classifier = 'javadoc'
+   from javadoc.destinationDir
+}
+
 publishing {
     publications {
         shadow(MavenPublication) { publication ->
             project.shadow.component(publication)
             artifact sourcesJar
         }
+        sonatype(MavenPublication) { publication ->
+            artifactId = "opendistro_sql_jdbc"
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
```

Comment:
at this time we don't have a very usable javadocs, infact executing the javadoc task fails currently due to cleanup needed in the javadoc comments. for now we can remove this.

Reply:
OK. I will remove it.

Reply:
I will add javadoc back. soantype forces you to have doc.